src/runtime_src/core/common/api/xrt_bo.cpp

```diff
@@ -185,6 +185,14 @@ class bo_impl
     size = prop.size;
   }
 
+  bo_impl(xclDeviceHandle dhdl, pid_t pid, xclBufferExportHandle ehdl)
+    : device(xrt_core::get_userpf_device(dhdl)), handle(device->import_bo(pid, ehdl)), free_bo(true)
+  {
+    xclBOProperties prop{};
```

Comment:
Learn something new today, didn't know we could do this.  

I always used the following syntax `xclBOProperties prop = {};`